server/index.js

```diff
@@ -7,13 +7,15 @@ import { PORT, startDBConnection } from './config';
 import logger from './config/winston';
 import routes from './routes';
 import { errorMiddleware } from './helpers/errorHandler';
+import rateLimiter from './config/rateLimit';
 
 const app = express();
 
 app.use(cors());
 app.use(bodyParser.urlencoded({ extended: false }));
 app.use(bodyParser.json());
 app.use(morgan('combined', { stream: winston.stream.write }));
+app.use(rateLimiter);
```

Comment:
Can we ensure that this is not used for test environment?

Reply:
Also, can we make the limits applicable to the API endpoints only. Something like:

```suggestion
app.use('/api', rateLimiter);
```
Please test to ensure that this works as expected.